docs/tutorials/index.rst

```diff
@@ -23,10 +23,6 @@ To get started with your first experiment, visit the :ref:`Quickstart for Model
 |                                 | you how to port a simple image classification model for the  |
 |                                 | MNIST dataset.                                               |
 +---------------------------------+--------------------------------------------------------------+
-| :doc:`pytorch-porting-tutorial` | The PyTorch porting tutorial provides helpful hints to       |
-|                                 | successfully integrate PyTorch code with the Determined      |
-|                                 | PyTorchTrial API.                                            |
-+---------------------------------+--------------------------------------------------------------+
```

Comment:
if we are keeping the tutorial, we'll need to keep it in the index page. if we are removing the tutorial, we can remove it from the index page.

Reply:
we'll also need to remove it from the toctree on this page.